ert3/algorithms/_sensitivity.py

```diff
@@ -0,0 +1,28 @@
+def _build_base_records(groups, parameters):
+    return {group_name: parameters[group_name].ppf(0.5) for group_name in groups}
+
+
+def one_at_the_time(parameters):
```

Comment:
Think type hinting would be very nice here!

Reply:
Hard to disagree! But do you agree that we should do that in a strategic PR or would you like me to introduce it already here?

Reply:
I think we can postpone it, but could also make sense to do it just for this function in this PR :slightly_smiling_face: